oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/Rules.java

```diff
@@ -41,6 +42,7 @@
     private List<CompositeAlarmRule> compositeRules;
     private DingtalkSettings dingtalks;
     private FeishuSettings feishus;
+    private WeLinkSettings weLinks;
```

Comment:
```suggestion
    private WeLinkSettings welinks;
```

I think this name in codes and config file make more sense.

Reply:
I'll think about it after google some demo.